src/dataSources/cloudFirestore/event.js

```diff
@@ -53,10 +53,13 @@ const event = dbInstance => {
     const docRef = dbInstance.doc(`${collectionName}/${id}`);
     const doc = await docRef.get();
 
-    return {
+    let result = {
       id: doc.id,
       ...doc.data(),
     };
+    if (!result.name) result = null;
```

Comment:
i'm confused what we're solving here. shouldn't we just check to see if we got a docId vs a name?

Reply:
//todo

Reply:
done. refactored to use doc.exists for check